docs/machine-learning/how-to-guides/how-to-use-the-automl-api.md

```diff
@@ -28,6 +28,7 @@ using Microsoft.ML.AutoML;
 
```

Comment:
on line 24, can you comment the ... like ///....

Reply:
Done